.goreleaser.yml

```diff
@@ -28,11 +28,7 @@ builds:
       - arm64
 
 archives:
-  - format: tar.gz
```

Comment:
let's use `zip` instead of `tar.gz`. `zip` works with cosmovisor.

Reply:
nice, I will try to run it locally with zip tarballs

Reply:
did not work :/ 
https://github.com/umee-network/umee-infra/pull/263#issuecomment-1250414590

Reply:
@robert-zaremba can you check this again?